packages/website/pages/docs/reference/prover-market.mdx

```diff
@@ -0,0 +1,24 @@
+# Prover Market
+
+## Use Prover
+
+Interact with provers directly with the prover market below. 
```

Comment:
we can leave a link to some swagger API docs generated by our own prover API. i believe @davidtaikocha can provide this.